src/collection-browser.ts

```diff
@@ -226,6 +236,10 @@ export class CollectionBrowser
   @query('infinite-scroller')
   private infiniteScroller!: InfiniteScroller;
 
+  private analyticsCategories = analyticsCategories;
```

Comment:
hmmm, thinking about usage. _who_ gets to declare the category - the component or the page the component is on? 
I like having a default like `collection-browser` but I think it should get set as a lit reactive property to receive the magic when we drop this component into the new collection page, or when its on a favorites page, etc.

suggestion: move to reactive lit property with default as `collection-browser`
this:
  - removes `analyticsCategories` object/enum
  - since the new property can be set with intention at the top-level, its name can be less generic, too.  like, "searchContext"
  - adds new property in demo like, `. searchContext=${'in-repo-demo'}`

```suggestion
```